src/metaschema/oscal_implementation-common_metaschema.xml

```diff
@@ -279,17 +279,17 @@
       </define-assembly>
       <define-assembly name="port-range">
             <formal-name>Port Range</formal-name>
-            <description>Where applicable this is the IPv4 port range on which the service operates.</description>
-            <define-flag name="start" as-type="non-negative-integer">
+            <description>Where applicable this is the transport layer protocol port range an IPv4-based or IPv6-based service uses.</description>
+            <define-flag name="start" as-type="nonNegativeInteger">
```

Comment:
Please update datatype `nonNegativeInteger` to `non-negative-integer`. We replaced the CamelCase datatypes.

Reply:
Reverted in https://github.com/usnistgov/OSCAL/pull/2070/commits/415600ab6fb5d8d10ff63e1a10ac2d7eb58712ac.